### ISSUE TITLE
Looking for paid help to install please look here

### ISSUE THREAD

**Issue by scatterp** [peatio/peatio#613]
Peatio is open source and you should not in the spirit of open source pay some one to install it for you
how ever if you are determined to go this route then I am willing to set you up
in return you can hire a Fiverr user to add some new features or fix some bugs 
that way you get your exchange PLUS extras 

leave your skype here if you need help

**Comment by MaffeProg** [comment]
You can also ask me for setting up your exchange + development/designing.

**Comment by zhanmusi** [comment]
I can set it up with customized blockchain and ico support.

**Comment by tonycontract** [comment]
@MaffeProg    @zhanmusi   

could you contact me? I need to hire a Developer to deploy and develop on Peatio.

I am also a developer, but I have no strong experience on RoR. So, I need help to quickly deploy and development some function. I want to evaluate the function and performance of this system.

My email is: tonyodeskcontract@gmail.com

Thanks a lot!

kind regards

**Comment by nomgbr** [comment]
I also need a programmer to set up Peatio
please contact me, my email and skype account is programmers30@gmail.com
Thanks!!

**Comment by scatterp** [comment]
nomgbr your skype account was not found could you check it ?

**Comment by tailongwang** [comment]
@tonycontract @nomgbr 
pls ping **live:apollo447722**
He setup 3 exchanges based on Peatio

**Comment by nomgbr** [comment]
@scatterp   In the end I managed to do it myself
Thank you anyway

**Comment by outbackdingo** [comment]
do you happen to have a good working repo of the code base.....

**Comment by scatterp** [comment]
you can see my repo if you like

**Comment by kamat-vg** [comment]
@scatterp I am about to buy a cloud hosting service to setup a bitcoin exchange. I need your help in setting up the exchange and would like to pay you for your time. Could you please get back to me at the earliest?

**Comment by outbackdingo** [comment]
Is your repo actually working ??? Im still loooking to roll out an
exchange......

**Comment by scatterp** [comment]
@kamat-vg  sent you mail
@outbackdingo  yes its working..

**Comment by outbackdingo** [comment]
not sure it is working, tried your install via the directions and bundle
exec rails server
 fails to start

**Comment by scatterp** [comment]
bundle exec rails server is not in the instructions 
you run the setup and then it says specifically "server is already running" 
so not sure what you did ...

**Comment by outbackdingo** [comment]
yeah found that in the doc though now cant login im getting a The network
connection is temporarily unavailable... Please refresh.
on the login page....

**Comment by scatterp** [comment]
some where you are far away from the instructions... 
what is your server specs?

**Comment by outbackdingo** [comment]
ubuntu 16.04 16Gb mem 750GB drive in a VM

**Comment by scatterp** [comment]
ok did you create the user deploy and usermod -a -G sudo deploy ?

**Comment by outbackdingo** [comment]
i ran Installer with minimal steps [￼(
https://github.com/scatterp/peatio/blob/master/install1.sh)]
wget https://raw.githubusercontent.com/scatterp/peatio/master/install1.sh
source install1.sh

**Comment by scatterp** [comment]
ok at the end whats it say ?
any errors during install?

**Comment by scatterp** [comment]
you have something small wrong not sure what

**Comment by outbackdingo** [comment]
ok, well it seems you want people to run install3.sh as user deploy which
isnt mentioned anywhere on the first page.... moving forward

**Comment by outbackdingo** [comment]
and still getting the exact same error

cant login im getting a The network connection is temporarily
unavailable... Please refresh.
on the login page....

**Comment by scatterp** [comment]
i dont know what code you are using but there is no install3.sh for quite some time 
please move this issue to http://github.com/scatterp/peatio and lets continue there include a skype address so we ca get this resolved quickly..

**Comment by outbackdingo** [comment]
install3.sh was a typo it was from install1.sh as user deploy
ive updated the ticket in your site under my issue with my skype address

**Comment by ovadiajon69** [comment]
Add me on Skype: ovadjon 
I'm having issues with my deployment and could use some help, I'm happy to pay.

**Comment by ovadiajon69** [comment]
Sorry the correct Skype ID is: ovadiajon

**Comment by scatterp** [comment]
done..

**Comment by sushilraghav** [comment]
I am looking for someone to install Peatio at my server. If successfully done i will also need to make some customizations. My skype: sushilraghav

**Comment by scatterp** [comment]
adding you now..

**Comment by sushilraghav** [comment]
I am having a vast experience in Peatio, Crypto Currency exchange and trading platforms with FIAT currency based on Bitcoin, litecoin, Ethereum, XRP etc, Core Wallets and Daemons. I am having expertise in ICO ie Initial Coin Offering full system development, White Paper creation with full architecture and high level documentation, smart contract writing and many more skills.

My Skype : sushilraghav and email is  admin@rensoftglobal.com

**Comment by wtfspiff** [comment]
hey scatterp 

can you add me on skype : sangam.gupta ( Location Dubai ) or email me at sangam@coinbech.com

Thanks
Sangam

**Comment by cashworld** [comment]
Hello everyone. I'm in need of someone who can help install peatio. Email is cashworldnet@gmail?com ..... thanks.

**Comment by HermantoSardan** [comment]
Hi scatterp,

Can you add my skype at herhmt@gmail.com, i am proceasing to setup bitcoin exchange on my location (Indonesia)

Need your service to install and customize peatio

Best regards,
Hermanto Sardan

**Comment by weesnaw1** [comment]
I don’t have skype but willing to pay to install. Abba.zabba@icloud.com

**Comment by mxo2** [comment]
we are professional team of blockchain and peatio customization/suppport...mail sam@mxo2.com or skype : physivert

**Comment by jhaibgl** [comment]
Hi scatterp,
Can you please add me?
Skype: jackyhai007@hotmail.com

**Comment by mxo2** [comment]
looking for customization for peatio add me on skype : physivert
